### DISCUSSION TITLE
Please add a timer also to your site, Cause its more exciting https://nylo.dev/learn-more/v6

### DISCUSSION THREAD

**Discussion by jitendravn** [nylo-core/nylo#171] [General] [answered]
Please add a timer also to your site, Cause its more exciting https://nylo.dev/learn-more/v6

**Comment by jitendravn** [comment]
@agordn52 I'm curious about nylo v6. Please update it soon

**Reply by agordn52** [reply, marked as answer]
@jitendravn,

When I've got more info I'll post an update on the website :)
It's almost done
